Documentation/AvailableOperations.md

```diff
@@ -0,0 +1,251 @@
+```
+
+#### Fetching Managers
+To fetch managers (parents or teachers) of a creator, use a `getManagers(...)` method:
+```Swift
+public func getManagers(userId userId: String?, pagingData: PagingData?, completion: UsersClosure?) -> RequestHandler
+```
+If you pass `nil` as a userId, managers of currently logged in user will be returned.
+
+#### Adding single Creator
+To add single Creator, use `newCreator(...)` method.
+```Swift
+public func newCreator(data creatorData: NewCreatorData,completion: UserClosure?) -> RequestHandler
+```
+
+#### Adding multiple Creators
+To add many Creators in single API call, use `createMultipleCreators(...)`.
+Note that this method will allow you to set amount of Creators and their birth year only. Usernames will be generated based on your nickname, but Creators should be able to change them after they login.
+```Swift
+public func createMultipleCreators(data data: CreateMultipleCreatorsData, completion: ErrorClosure?) -> RequestHandler
+```
+
+#### Editing profile
+To edit profile, use `editProfile(...)` method.
+```Swift
+public func editProfile(userId identifier: String, data: EditProfileData, completion: ErrorClosure?) -> RequestHandler
+```
+For data which can be edited see [Account Details](https://partners.creatubbles.com/api/#account-details) section of API Reference.
+
+### Gallery management
+#### Fetching single gallery
+To fetch single gallery, use `getGallery(galleryId ...)` method.
+```Swift
+public func getGallery(galleryId galleryId: String, completion: GalleryClosure?) -> RequestHandler
+```
+
+#### Fetching galleries by creation
+To fetch galleries by creation, use `getGalleries(creationId ...)` method.
+```Swift
+public func getGalleries(creationId creationId: String, pagingData: PagingData?, sort: SortOrder?, completion: GalleriesClosure?) -> RequestHandler
+```
+In completion closure you will receive galleries which contains selected creation.
+
+#### Fetching galleries by user
+To fetch galleries by user, use `getGalleries(userId ...)` method.
+```Swift
+public func getGalleries(userId userId: String?, pagingData: PagingData?, sort: SortOrder?, completion: GalleriesClosure?) -> RequestHandler
+```
+In completion closure you will receive galleries owned by selected user. If you pass `nil` as a userId, you will receive galleries of currently logged in user.
+
+#### Fetching owned galleries
+To fetch galleries owned by currently logged in user, use `getMyOwnedGalleries(...)` method.
+```Swift
+public func getMyOwnedGalleries(pagingData: PagingData?, completion: GalleriesClosure?) -> RequestHandler
+```
+You can also fetch all owned galleries in bach mode: `getOwnedGalleriesInBatchMode(...)`
+```Swift
+public func getOwnedGalleriesInBatchMode(completion: GalleriesBatchClosure?) -> RequestHandler
+```
+
+#### Fetching galleries shared with currently logged in user
+To fetch galleries owned shared with currently logged in user, use `getMySharedGalleries(...)` method.
```

Comment:
'To fetch galleries owned shared...'
You probably meant just "shared", not owned.